### ISSUE TITLE
Publish button dropdown breaks to next line

### ISSUE THREAD

**Issue by igodorogea** [neos/neos-development-collection#1715]
When clicking on "Publish" button dropdown, the arrow section of the button jumps bellow the button, thus extending the parent box area and overflow the buttons bellow it, making them inaccessible.

![Publish button dropdown breaks](https://user-images.githubusercontent.com/4079076/31169765-6a475334-a902-11e7-9150-11299421eae6.jpg)

**Comment by bwaidelich** [comment]
@igodorogea Thanks for reporting this issue. In order for us to reproduce this, can you please share what version of Neos you are using?

**Comment by igodorogea** [comment]
@bwaidelich It's Neos 3.2.0.
However I noticed that behavior appears only on my Surface Pro 4(2736x1824 which is a HiDPI display).
On a Full HD display I wasn't able to reproduce.
In Dev Tools I noticed that each time the dropdown arrow is clicked - a new div is inserted between publish button and dropdown button(see the attached image).

<a href="https://user-images.githubusercontent.com/4079076/31442186-3a171950-ae9e-11e7-8371-45f4db3c79c7.jpg" target="_blank">Img</a>
